simapp/simd/cmd/commands.go

```diff
@@ -38,10 +40,20 @@ func initRootCmd(
 	interfaceRegistry codectypes.InterfaceRegistry,
 	appCodec codec.Codec,
 	basicManager module.BasicManager,
+	extraInfo version.ExtraInfo,
 ) {
 	cfg := sdk.GetConfig()
 	cfg.Seal()
 
+	type contextKey struct {
```

Comment:
The struct should be in the version package, and does not need to contain any key.